src/internal/connector/graph_connector_test.go

```diff
@@ -66,7 +66,7 @@ func (suite *GraphConnectorIntegrationSuite) TestSetTenantUsers() {
 
 	newConnector.graphService = *service
 
-	suite.Equal(len(newConnector.Users), 0)
+	suite.Equal(0, len(newConnector.Users))
```

Comment:
nit
```suggestion
	suite.Empty(newConnector.Users)
```

Reply:
All this well curated fanciness to keep track of.